### PR TITLE
Allow DataFrames.jl 1.x

### LINKED ISSUE #18: Is it possible to add support for DataFrames.jl 1.x release?
If it is possible it would be great to have it (and could you please make a release?). Thank you!

### PR DESCRIPTION
**Pull request by bkamins**
Fixes https://github.com/davidavdav/ROCAnalysis.jl/issues/18

### CODE REVIEW

**Comment by bkamins** [comment]
@davidavdav - if you accepted this PR could you please make a release? Thank you!

**Comment by davidavdav** [comment]
@juliaregistrator register

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "ROCAnalysis"
 uuid = "f535d66d-59bb-5153-8d2b-ef0a426c6aff"
 repo = "https://github.com/davidavdav/ROCAnalysis.jl.git"
-version = "0.3.2"
```
Comment (davidavdav) [inline Project.toml:4]: @juliaregistrator register

**Comment by bkamins** [comment]
Thank you!

**Comment by bkamins** [comment]
For some reason a release was not made automatically in this repository (but the package can be correctly installed in 0.3.3 version)

**Pull request merged**